docs/package/gluon-ssid-changer.md

```diff
@@ -0,0 +1,70 @@
+gluon-ssid-changer
+==================
+
+This package adds a script to change the SSID to an Offline-SSID when there is
+no connection to any gateway.
+This SSID can be generated from the nodes hostname with the first
+and last part of the nodename or the mac address, to allow observers to 
+recognise which node is down. The script is called once a minute by micron.d
+and it will change from online to offline-SSID maximum once every (definable)
+timeframe.
+
+You can enable/disable it in the config mode.
+
+Once every timeframe it checks if there's still a gateway reachable. There can be
+selected two algorithms to analyze if a gateway is reacheable:
+
+- `tq_limit_enabled=true`: (not working with BATMAN_V) define an upper and lower
+  limit to turn the offline_ssid on and off in-between these two values the SSID
+  will never be changed to prevent it from toggeling every minute.
+- `tq_limit_enabled=false`: there will be only checked, if the gateway is
+  reachable with:
+
+        batctl gwl -H
+
+Depending on the connectivity, it will be decided if a change of the SSID is 
+necessary: There is a variable `switch_timeframe` (for ex.  1440 = 24h) that 
```

Comment:
Leave this out "Depending on the connectivity, it will be decided if a change of the SSID is 
necessary"